lib/foreman_discovery/engine.rb

```diff
@@ -9,6 +9,7 @@ module ForemanDiscovery
   class Engine < ::Rails::Engine
 
     # support pre-4 Rails versions
+    config.autoload_paths += Dir["#{config.root}/app/controllers/foreman_discovery/concerns"]
     config.autoload_paths += Dir["#{config.root}/app/controllers/concerns"]
```

Comment:
Just because a 'HostsControllerExtensions' without scoping it in ForemanDiscovery seemed too generic and possibly another plugin would use that.